webapp/src/webapp/connections/views/connection_connect.cljs

```diff
@@ -133,79 +127,51 @@
     [:div {:class "flex justify-end gap-3"}
      [button/secondary {:text "Minimize"
                         :outlined true
-                        :on-click #(minimize-modal draggable-card-content)}]
+                        :on-click #(minimize-modal)}]
      [button/red-new {:text "Disconnect"
                       :on-click #(close-connect-dialog)}]]]])
 
-(defn- loading [connection]
-  [:section
-   [:div {:class "max-w-7xl px-4"}
-    [:div {:class "flex flex-col items-center gap-regular justify-center h-full my-x-large"}
-     [:div {:class "flex gap-small"}
-      [loaders/simple-loader]
-      [:span {:class "text-gray-600"}
-       "Establishing connection with:"]]
-     [:span {:class "font-bold text-gray-600"}
-      (:name (:data connection))]]]])
+(defn- loading []
+  [:> Flex {:direction "column" :gap "5"}
+   [:> Box {:class "space-y-radix-3"}
+    [:> Heading {:size "6" :as "h2"}
+     "Hoop Access"]
+
+    [:> Box {:class "space-y-radix-2"}
+     [:> Skeleton {:height "20px"}]
+     [:> Skeleton {:height "20px"}]]]
+
+   [:> Box {:class "space-y-radix-3"}
+    [:> Skeleton {:height "60px"}]
+    [:> Skeleton {:height "60px"}]
+    [:> Skeleton {:height "60px"}]
+    [:> Skeleton {:height "60px"}]]])
 
 (defn- failure [connection]
-  [:main {:class "p-2"}
+  [:main
    [:header {:class "mb-2"}
-    [h/h3 "Hoop access details"]]
-
-   [:section {:class "flex gap-small"}
-    [:div {:class "flex items-center gap-small my-small"}
-     [:> hero-micro-icon/XMarkIcon {:class "h-4 w-4 text-red-500"}]
-     [:span {:class "text-xs text-gray-700"}
-      "Failed attempt to connect to:"]
-     [:span {:class "font-bold text-xs text-gray-600"}
-      (:connection_name (:data connection))]]]
+    [:> Heading {:size "6" :as "h2"}
+     "Hoop Access"]]
 
    [:section {:class "my-regular"}
     [:span {:class "text-sm text-gray-800"}
-     "Make sure hoop app is open and running with a valid account logged in."]
-    [:span {:class "block text-nowrap text-sm text-gray-800"}
-     "You can also download hoop.dev desktop app and setup your hoop access."]]
+     "Download and open Hoop Desktop App to enable a direct access to this connection."]]
```

Comment:
quick adjust for better semantics in the phrase: "Download and open Hoop Desktop App to have direct access to this connection."